js/backend.js

```diff
@@ -0,0 +1,30 @@
+const getData = (onSuccess) => {
+  fetch('https://23.javascript.pages.academy/keksobooking/data')
+    .then((response) => response.json())
+    .then((similarAds) => {
```

Comment:
```suggestion
    .then((data) => {
```

это общая функция, поэтому параметры тут абстрактны